src/audio/mux/mux_ipc4.c

```diff
@@ -105,7 +105,7 @@ static void set_mux_params(struct processing_module *mod)
 		struct ipc4_audio_format *audio_fmt;
 
 		comp_dev_for_each_producer(dev, source) {
-			j = buf_get_id(source);
+			j = IPC4_SINK_QUEUE_ID(buf_get_id(source));
```

Comment:
just to check - it does have to be `IPC4_SINK_QUEUE_ID()`, not `IPC4_SRC_QUEUE_ID()`?